cpp/modmesh/base.hpp

```diff
@@ -131,43 +132,6 @@ class SpaceBase : public NumberBase<I, R>
 
 }; /* end class SpaceBase */
 
-// Taken from https://stackoverflow.com/a/12262626
-class Formatter
```

Comment:
We don't need this temporary workaround now.

Reply:
I can do it. However, I suggest only do this for the formater. If we want to add all backward capacity for all C++23 features, it's actually hard to use new features.

Reply:
I only care about the class `Formatter`.  It is example code.  I don't need it to work as a fall-back.  I only want it to be still working in the repository.

What about keeping it in a gtest and comparing the results with `std::format`?